string-manipulation-2/src/truncate.js

```diff
@@ -0,0 +1,25 @@
+/* exported truncate */
+function truncate(length, string) {
+  var cut = '';
+  if (length < string.length) {
+    for (var i = 0; i < length; i++) {
+      cut += string[i];
+    }
+  } else {
+    for (var j = 0; j < string.length; j++) {
+      cut += string[j];
+    }
```

Comment:
In this case, you want to store the entire string, so this can be simplified to `cut = string`